tests/e2e/testsuites/dynamically_provisioned_delete_pod_tester.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testsuites
+
+import (
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/tests/e2e/driver"
+	. "github.com/onsi/ginkgo"
+	"k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+// DynamicallyProvisionedDeletePodTest will provision required StorageClass and Deployment
+// Testing if the Pod can write and read to mounted volumes
+// Deleting a pod, and again testing if the Pod can write and read to mounted volumes
+type DynamicallyProvisionedDeletePodTest struct {
```

Comment:
I didn't find where do we run this test case, and im not sure why do we need to test it through deployment. For the sake of limiting the scope of this PR, can we hold off on this test case?

Reply:
The file was probably collapsed, here is the test: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/173/files#diff-4b7e52cf33a96cc1c8a4d9974c34027aR315

I would prefer to keep this test as its actually pretty important, this is the only one thats tests the scenario of what happens when a pod restarts and the volume gets remounted, it tests the detach/attach flow and validates that the volume is not being formatted after the second attach.